nginx-base/alpine-nginx/Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM nginx:stable-alpine
 
-ENV PARSE_TEMPLATE_VERSION=v1.0.0 \
-    PARSE_TEMPLATE_HASH=8d1dc39e701b938f4874f3f8130cd3a324e7fa4697af36541918f9398dd61223
+ENV PARSE_TEMPLATE_VERSION=v1.0.1 \
+    PARSE_TEMPLATE_HASH=dcfbc9e843292664aab43cd942694019ae566dc6ab83567f4339e942da76360b
 
 RUN set -exu \
  && apk update \
  && apk upgrade --available \
  && apk add --virtual build-deps curl \
- && curl -L -o /usr/bin/parse-template https://github.com/cocreators-ee/parse-template/releases/download/${PARSE_TEMPLATE_VERSION}/parse-template-linux-amd64 \
+ && curl -L -o /usr/bin/parse-template https://github.com/cocreators-ee/parse-template/releases/download/${PARSE_TEMPLATE_VERSION}/parse-template_Linux_x86_64 \
```

Comment:
Am I missing something or will this install the x86_64 version on ARM based images as well?